lib/Cleantalk/Common/Validate.php

```diff
@@ -80,9 +80,21 @@ public static function isValidFilePath($variable)
         // TODO
     }
 
-    public static function isUrl($variable)
+    public static function isUrl($url)
     {
-        // @ToDo
-        return $variable;
+        return ( strpos($url, 'http://') !== false || strpos($url, 'https://') !== false ) &&
+               filter_var($url, FILTER_VALIDATE_URL);
+    }
+
+    /**
+     * Checks if given string is valid regular expression
+     *
+     * @param string $regexp
+     *
+     * @return bool
+     */
+    public static function isRegexp($regexp)
+    {
+        return @preg_match('/' . $regexp . '/', '') !== false;
```

Comment:
Better to @ delimiter here, to avoid conflicts for cases when delimiters are passed  to the function.

Reply:
Everything is ok!